cmd/soroban-cli/src/commands/contract/init.rs

```diff
@@ -1,12 +1,12 @@
-use std::fs::read_to_string;
-use std::path::Path;
-use std::{env, fs, io};
-
 use clap::builder::{PossibleValue, PossibleValuesParser, ValueParser};
 use clap::{Parser, ValueEnum};
+use rust_embed::RustEmbed;
 use serde::Deserialize;
+use std::fs::read_to_string;
```

Comment:
I personally like all std imports at the top, then external crates and then workspace/crate. Also the imports can be combined. 
```rust
use std::{fs::..., path::Path, ...}
```

Reply:
I'll get a fix pushed up for this shortly!

Also, in general, do you prefer to have external crates imported at the top of the file with `use`, over qualifying them in-line? I've realized I have a bit of both here, and am thinking that some of the qualified imports could be updated.